### ISSUE TITLE
Axios Post Network Error

### ISSUE THREAD

**Issue by criss02-cs** [adrianhajdin/project_medical_pager_chat#20]
Hi, i was just seeing your video, but when i try to sign up it gave me an Network error with Axios
![image](https://user-images.githubusercontent.com/52246268/178153285-6727ab50-9a53-47f8-94c9-1727424b9fd2.png)

**Comment by Adityakumar-blip** [comment]
I'm getting the same error.

**Comment by avigyanRoy** [comment]
Facing the same issue. Would be great if it is resolved soon.

**Comment by TheReaLLsQuaLL** [comment]
To solve this issue on the auth.jsx make the const URL like this

const URL = "http://localhost:5000/auth";

**Comment by YongkangLu** [comment]
POST http://localhost:5000/auth/signup 404 (Not Found)

any help on this whenever try to click sign up it throws the above

**Comment by TheReaLLsQuaLL** [comment]
Can you add your code so maybe can help better.Also you can check my repo if you want to see working code

**Comment by pwn-mm** [comment]
> Facing the same issue. Would be great if it is resolved soon.

This error is probably because of React version. Please downgrade to React17 and try again.

**Comment by pwn-mm** [comment]
> To solve this issue on the auth.jsx make the const URL like this
> 
> const URL = "http://localhost:5000/auth";

Yes yes, this one works.

**Comment by jahanvi1903** [comment]
> > To solve this issue on the auth.jsx make the const URL like this
> > const URL = "http://localhost:5000/auth";
> 
> Yes yes, this one works.



after changing i got another error
![Screenshot (35)](https://github.com/adrianhajdin/project_medical_pager_chat/assets/113800978/ee7ff73f-d168-4f99-848d-7027d1786226)

**Comment by TheReaLLsQuaLL** [comment]
> > > To solve this issue on the auth.jsx make the const URL like this
> > > const URL = "http://localhost:5000/auth";
> > 
> > 
> > Yes yes, this one works.
> 
> after changing i got another error ![Screenshot (35)](https://user-images.githubusercontent.com/113800978/266837635-ee7ff73f-d168-4f99-848d-7027d1786226.png)

Did you solve it yet or having the same problem?

**Comment by amish0301** [comment]
Yes, I'm also getting the same error. i tried all possible ways like changes URL from HTTP <-> HTTPS then I used scryptSync for generating random User_id but still getting the same Error. 

if anyone Solved please share your ans.
